compaction/compaction_group_view.hh

```diff
@@ -38,8 +38,8 @@ public:
     // min threshold as defined by table.
     virtual unsigned min_compaction_threshold() const noexcept = 0;
     virtual bool compaction_enforce_min_threshold() const noexcept = 0;
```

Comment:
> replica: Futurize retrieval of sstable sets in compaction_group_view
>
> This will allow upcoming work to gently produce a sstable set for
each compaction group view. Example: repaired and unrepaired.

Not sure it's needed. With tablets, we have a relatively small number of sstables per tablet.

Reply:
Looks like the cause for this is the fact that you don't have separate sstable sets for repaired/unrepaired.

If the only user is compaction/repair, it's not too bad. Compaction/repair also insert sstables, which is a slow/futurized process. But the data path should not use these. Does it?

Reply:
I guess it doesn't, since this is for compaction_group_view.